src/components/mobile-payment-link/MobilePaymentLink.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="mobile-payment-link">
+    <div class="header">
+      <span class="headerText">Payment link</span>
+      <span class="copyButton" @click.stop.prevent="copyLink">Copy</span>
+    </div>
+    <a class="link" :href="link">{{ link }}</a>
+    <input type="hidden" id="linkToCopy" :value="link" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: "mobile-payment-link",
+  props: {
+    link: { type: String, default: "" }
+  },
+  methods: {
+    copyLink() {
+      let textToCopy = document.querySelector("#linkToCopy");
+      textToCopy.setAttribute("type", "text");
+      textToCopy.select();
+      document.execCommand("copy");
+      textToCopy.setAttribute("type", "hidden");
+      window.getSelection().removeAllRanges();
```

Comment:
Is this working in all browsers we want to support? Afaik, copy pasting had some cross-browser problems until recently?

Reply:
Here is compatibility as of today, shall I change the code for any browser?

<img width="1000" alt="Screen Shot 2019-09-09 at 12 44 35 PM" src="https://user-images.githubusercontent.com/41395780/64524804-a5b35e80-d2ff-11e9-9933-ae1a3def106a.png">

Reply:
That seems better than I remembered. Should be fine indeed